src/assets/js/event-factory.ts

```diff
@@ -0,0 +1,13 @@
+import { TabulatorComponent } from "@/entry";
```

Comment:
That import create a **Dependency cycle**, because the `TabulatorComponent` import the `event-factory`. 
Maybe you can change the `TabulatorComponent` for the `Vue`. You only need a reference for the `$emit` function. Please, provide a test for that function and move that file to another folder. The assets is just only static files, you can create another folder, something like **feature/events** and put the related files together